tfhe/src/integer/server_key/radix_parallel/add.rs

```diff
@@ -719,7 +719,7 @@ impl ServerKey {
         // overhead of dynamic dispatch is negligible compared to multithreading, PBS, etc.
         // we defer all calls to a single implementation to avoid code bloat and long compile
         // times
-        #[allow(clippy::type_complexity)]
+        #[allow(clippy::items_after_statements)]
```

Comment:
🤔

Reply:
Function definition and constants in other functions must be first
`#[allow(clippy::type_complexity)]` was useless